### DISCUSSION TITLE
用yarn start启动的生产环境服务，直接报500错

### DISCUSSION THREAD

**Discussion by zyhwyl** [eggjs/egg#5297] [Q&A]
![image](https://github.com/eggjs/egg/assets/4638740/528832b0-eca4-4858-8a42-0daa9d4d9d6e)
如图，看了下日志报错是nodejs.Error: Please set config.keys first
不过根本原因是，这样方式启动没有读取目录的config文件，能否看看如何解决下

**Comment by killagu** [comment]
按照提示在 config.default.js 中配置 keys 就可以了。可以参考文档 https://www.eggjs.org/zh-CN/core/cookie-and-session#cookie-%E7%A7%98%E9%92%A5

**Reply by zyhwyl** [reply]
这里加了，没用，用这个方式启动的服务都不读取config.default.js中的配置，用yarn dev启动的是可以的

**Reply by zyhwyl** [reply]
脚手架创建的demo项目就有问题，你可以试试
